components/Carousel/Carousel.js

```diff
@@ -1,14 +1,46 @@
 class Carousel {
+	constructor(element) {
+		this.element = element;
+		this.leftBtn = this.element.querySelector('.left-button');
+		this.rightBtn = this.element.querySelector('.right-button');
+		this.imgs = this.element.querySelectorAll('img');
+		this.index = 1;
 
+		this.leftBtn.addEventListener('click', () => this.left());
+		this.rightBtn.addEventListener('click', () => this.right());
+	}
+	left() {
+		this.imgs.forEach((img) => (img.style.display = 'none'));
+
+		if (this.index > -1) {
+			this.imgs[this.index].style.display = 'block';
+			this.index -= 1;
+		} else {
+			this.imgs[3].style.display = 'block';
+			this.index = 3;
```

Comment:
the whole idea of component is to make it reusable. when you set the index to 3. You are limiting the number of items in this carousel.